src/models/room.ts

```diff
@@ -1217,9 +1224,6 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
             // Set our new fresh timeline as the live timeline to continue syncing
             // forwards and back paginating from.
             timelineSet.setLiveTimeline(newTimeline!);
-            // Fixup `this.oldstate` so that `scrollback` has the pagination tokens
-            // available
-            this.fixUpLegacyTimelineFields();
```

Comment:
The comment is indeed stale but I think we still need to update the re-emitter whenever the live timeline changes

Reply:
yes this is definitly required. (Tests without updating the reemitter breaks a bunch of things)